narwhal/primary/src/primary.rs

```diff
@@ -180,6 +186,16 @@ impl Primary {
             &primary_channel_metrics,
         ));
 
+        // Convert authority private key into key used for random beacon.
+        let randomness_private_key = fastcrypto::groups::bls12381::Scalar::from_byte_array(
+            signer
+                .copy()
+                .private()
+                .as_bytes()
+                .try_into()
+                .expect("key length should match"),
+        )
+        .expect("should work to convert BLS key to Scalar");
```

Comment:
is this a temporary measure until we add another key? I thought we didn't want to re-use signing keys as encryption keys?